PPP.py

```diff
@@ -265,8 +282,16 @@ def setupVariables():
             local_convert = 'u2w'
 
     playlist = [convertPath(track, local_convert, False) for track in playlist]
+    
 
-    local_prepend = os.path.commonpath(playlist)
+    if local_unix == ppp_unix:  # Same machine type
+        local_prepend = os.path.commonpath(playlist)
+    elif local_unix:            # On Windows, looking at a Unix sytle playlist
+        import posixpath
+        local_prepend = posixpath.commonpath(playlist)
+    else:                       # On Unix, looking at a Windows sytle playlist
+        import ntpath
+        local_prepend = ntpath.commonpath(playlist).replace("/","\\\\")
```

Comment:
What is the need for `.replace("/","\\\\")`?
Shouldn't a Windows style playlist already have backslashes not forward?

Reply:
This came in while hacking around on getting commonpath working, and getting the Windows local_prepend to work.  Further validation and debug can perhaps prove this unnecessary.